src/main/webapp/profile.js

```diff
@@ -13,6 +13,11 @@
 // limitations under the License.
 
 // Allows certain fields in the profile to be edited, hides edit button, and displays update button. 
+
+function onLoad() {
```

Comment:
Since we're incorporating more than one js file per HTML now it would make sense to add the page's name to each onload() so we don't have any duplicate functions in the future. And in the body of every HTML page we can have `onload=onload[page name]()`

Reply:
Done